packages/apps/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -563,9 +564,10 @@ export const SpacePlugin = ({
               return intentPlugin?.provides.intent.dispatch({
                 action: LayoutAction.OPEN_POPOVER,
                 data: {
-                  anchorId: `dxos.org/ui/navtree/${intent.data.object.id}`,
+                  anchorId: `dxos.org/ui/${intent.data.caller}/${intent.data.object.id}`,
```

Comment:
should the caller just pass the whole anchor id that it wants it anchored to?

Reply:
It’s a good question… I’m actually not sure. That seems like it would have a different set of consequences & use-cases.